package.json

```diff
@@ -8,7 +8,7 @@
     "test": "jest",
     "test-watch": "nodemon --exec 'npm-run-all dist test' --watch ./tokens --watch ./formats --watch ./gulpfile.js --watch ./scripts",
     "clean": "rimraf ./dist",
-    "getTokensFromInvision": "node ./scripts/getTokensFromInvision.js",
+    "getTokensFromInvisionDSM": "node ./scripts/getTokensFromInvisionDSM.js",
```

Comment:
Can use use kebab-case (get-tokens-from-invision-dsm) for package scripts?

I think it's a bit easier to type, easier to autocomplete if you haven't turned off case sensitivity and we already do that for `dist-ci`